documentation/src/main/asciidoc/server_guide/security.adoc

```diff
@@ -84,18 +84,101 @@ The following configuration enables authentication against ApplicationRealm, usi
 </hotrod-connector>
 ----
 Notice the server-name attribute: it is the name that the server declares to incoming clients and therefore the client configuration must match.
+You can specify multiple mechanisms and they will be attempted in order.
 
-Once you have configured a secured Hot Rod connector, you can connect to it using the Hot Rod client:
+==== SASL Quality of Protection
 
-.Hot Rod client configuration
+Some mechanisms also support integrity and privacy protection, also known as Quality
+of Protection (or qop): during authentication negotiation, ciphers are exchanged between client and server, and they can be used to add checksums and encryption to all subsequent
+traffic. The possible
```

Comment:
Looks like you forgot to remove `The possible`.  Also the previous line should be split up